central/namespace/datastore/datastore.go

```diff
@@ -152,7 +152,7 @@ func (b *datastoreImpl) AddNamespace(ctx context.Context, namespace *storage.Nam
 	if err := b.store.Upsert(ctx, namespace); err != nil {
 		return err
 	}
-	return b.indexer.AddNamespaceMetadata(namespace)
+	return nil
 }
```

Comment:
suggestion is annoying me so I will just comment them.

return b.store.Upsert(ctx, namespace)